addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -216,7 +217,12 @@ function factory(dependencies) {
              * @returns {integer}
              */
             const _getNextIndex = index => {
-                const directionOffset = reverse ? -1 : 1;
+                let directionOffset = 1;
+                if (textDirection === 'ltr') {
+                    directionOffset = reverse ? 1 : -1;
+                } else {
+                    directionOffset = reverse ? -1 : 1;
+                }
```

Comment:
- just `let directionOffset;`, initial value `1` is useless
- variable `textDirection` not necessary, can simply use `this.env.messaging.locale.textDirection` direction in `if` conditional expression.